plugin-script-python/src/main/java/io/kestra/core/tasks/scripts/Python.java

```diff
@@ -146,20 +148,39 @@ public class Python extends AbstractBash implements RunnableTask<ScriptOutput> {
     @Builder.Default
     protected Boolean virtualEnv = true;
 
+    @Schema(
+        title = "Package manager for Python dependencies",
+        description = "Package manager to use for installing Python dependencies. " +
+            "Options: 'pip' (default), 'uv'. " +
+            "UV automatically falls back to pip if not available.",
+        allowableValues = {"PIP", "UV"}
+    )
+    @PluginProperty
+    @Builder.Default
+    protected Property<PackageManagerType> packageManager = Property.ofValue(PackageManagerType.PIP);
+
     protected String virtualEnvCommand(RunContext runContext, List<String> requirements) throws IllegalVariableEvaluationException {
         List<String> renderer = new ArrayList<>();
 
         if (runContext.render(this.exitOnFailed).as(Boolean.class).orElseThrow()) {
             renderer.add("set -o errexit");
         }
-        renderer.add(this.pythonPath + " -m venv --system-site-packages " + workingDirectory + " > /dev/null");
 
-        if (requirements != null) {
-            renderer.addAll(Arrays.asList(
+        if (PackageManagerType.UV.equals(runContext.render(packageManager).as(PackageManagerType.class).orElse(PackageManagerType.PIP))) {
```

Comment:
Extract this to a variable for readability: `runContext.render(packageManager).as(PackageManagerType.class).orElse(PackageManagerType.PIP)`